services/tenant-ui/frontend/src/components/innkeeper/reservation/RefreshPassword.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <Button
+    :label="$t('reservations.refreshPassword')"
+    class="p-button-rounded p-button-text"
+    icon="pi pi-refresh"
+    @click="refresh"
+  />
+</template>
+
+<script setup lang="ts">
+// Vue
+import { ref } from 'vue';
+// PrimeVue / etc
+import Button from 'primevue/button';
+import Dialog from 'primevue/dialog';
+import { useToast } from 'vue-toastification';
+// State
+import { useInnkeeperTenantsStore } from '@/store';
+import { storeToRefs } from 'pinia';
+import InputText from 'primevue/inputtext';
+const innkeeperTenantsStore = useInnkeeperTenantsStore();
+const { loading } = storeToRefs(useInnkeeperTenantsStore());
+import { useConfigStore } from '@/store';
+import { useI18n } from 'vue-i18n';
+const { config } = storeToRefs(useConfigStore());
+const toast = useToast();
+const { t } = useI18n();
```

Comment:
Would organize these a bit more, keeping the use*Store ones together and the storetorefs ones next to each other, the primevue imports next to each other etc